gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -841,37 +841,41 @@ function TellTraitorsAboutTraitors()
 		end
 	end
 
-	hook.Run("TTT2TellTraitors", traitornicks)
+	for _, v in ipairs(player.GetAll()) do
+		if v:HasTeam(TEAM_TRAITOR) then
 
-	-- This is ugly as hell, but it's kinda nice to filter out the names of the
-	-- traitors themselves in the messages to them
-	local traitornicks_min = #traitornicks < 2
+			local tmp, shouldShow = table.Copy(traitornicks), true
```

Comment:
you don't need to init shouldShow, the `if shouldShow == false` doesn't trigger if it's nil or true